docs/docs/services/qix-engine/doc-synchronization.md

```diff
@@ -6,45 +6,22 @@
 
 ## Description
 
-For sharing documents between multiple instances the Qlik Associative Engine uses a message broker called [RabbitMQ](https://www.rabbitmq.com/).
+For sharing documents between multiple instances the Qlik Associative Engine uses file polling to detect document changes.
+If there are e.g. changes to the data blob or generic objects being created/removed in a document,
+this would trigger a notification to other engine instances sharing the same document.
+The file polling interval for checking for document changes is every 10 seconds.
+
+When a document change has been detected the client will receive a push message in the outdated session,
+and can act on it as preferred.
+There is an example in `enigma.js` for how to subscribe on these events [here](https://github.com/qlik-oss/enigma.js/blob/master/docs/api.md#event-changed).
+
 With this feature the Qlik Associative Engine instances running in a cluster, regardless of orchestration,
```

Comment:
I'd like for us to be clear here what it supports, e.g. `This feature support changes based on creation, editing, and deletion of generic objects, as well as reload of the data.`